src/client/components/app/App.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
I think this file should be App.test.jsx.  Sorry if I missed this on the Chart component PR earlier :(